navfn/src/navfn_ros.cpp

```diff
@@ -346,15 +346,17 @@ namespace navfn {
       return;
     }
 
+    //given an empty path we won't do anything
+    if(path.empty()) {
```

Comment:
This should not return, we still want to publish the empty path.

The ticket filed talks about not having the timestamp / frame. But you still have to have it published, which happens below, to get any result.

Reply:
Ah, okay. I thought the purpose of this topic was purely for visualization, in which case it doesn't make sense to visualize an empty path (therefore we would revert the code to the behavior in 46db461a98a1832a2d1b98b9d6f76f83dee12d63). But if it's the expectation that this topic returns the result of planning, failure included, that makes sense.